desktop_version/.gitignore

```diff
@@ -16,3 +16,6 @@ VVVVVV
 # Game data
 data.zip
 leo.ogg
+
+# Ignore any swap file
+*.swp
```

Comment:
Ok, but could you instead configure your editor to not place swap files inside the working directory?

Reply:
```vim
set directory=$HOME/.vim/swapfiles//
```
(double `//` intentional, create that directory first)

Reply:
(or just use neovim, which has this by default)